src/bentoml/_internal/gradio.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import typing as t
+from pathlib import Path
+
+from bentoml.exceptions import MissingDependencyException
+
+R = t.TypeVar("R")
+
+try:
+    import gradio as gr
+    from gradio import Blocks
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        """'gradio' is required by 'bentoml.gradio', run 'pip install -U gradio' to install gradio.""",
+    )
+
+
+def mount_gradio_app(
+    blocks: Blocks,
+    path: str,
+    name: str = "gradio_ui",
+):
+    from _bentoml_sdk.service import Service
+    from bentoml._internal import server
```

Comment:
we don't call the server from here anymore right? it should be `_bentoml_impl.server` iirc.

Reply:
+1